Cubical/Data/Nat/GCD.agda

```diff
@@ -0,0 +1,116 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Data.Nat.GCD where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
+
+open import Cubical.Induction.WellFounded
+
+open import Cubical.Data.Fin
+open import Cubical.Data.Sigma as Σ
+open import Cubical.Data.Prod hiding (_×_) renaming (_×Σ_ to _×_)
+open import Cubical.Data.NatPlusOne
+
+open import Cubical.HITs.PropositionalTruncation as PropTrunc
+
+open import Cubical.Data.Nat.Base
+open import Cubical.Data.Nat.Properties
+open import Cubical.Data.Nat.Order
+open import Cubical.Data.Nat.Divisibility
+
+private
+  variable
+    m n d : ℕ
+
+-- common divisors
+
+isCD : ℕ → ℕ → ℕ → Type₀
+isCD m n d = (d ∣ m) × (d ∣ n)
+
+isPropIsCD : isProp (isCD m n d)
+isPropIsCD = isPropΣ isProp∣ (λ _ → isProp∣)
```

Comment:
You can use https://github.com/agda/cubical/blob/master/Cubical/Foundations/HLevels.agda#L230 (this lemma should be renamed)

Reply:
I renamed it and moved it to `Data.Prod.Properties`.